universum/main.py

```diff
@@ -74,17 +77,17 @@ def execute(self):
         self.launcher.launch_project()
         if afterall_configs:
             if not self.settings.no_diff:
-                if self.vcs.supports_copy_cl_files_and_revert():
+                try:
```

Comment:
This seems like a more pythonic approach

Reply:
What if `launch_custom_configs` raises NotImplementedError exception?

Reply:
then previously that would stop the execution - > now it would just get looged as an unsupported action

Reply:
My point is that call to `launch_custom_configs` must be moved out of the try...except block, because with this change you log everything the same way.
Launcher should not be allowed to raise NotImplementedError, and the resulting error message must be different.